frontend/talentsearch/src/js/components/search/SearchForm.tsx

```diff
@@ -482,13 +496,13 @@ const SearchForm = React.forwardRef<SearchFormRef, SearchFormProps>(
                   }),
                 },
                 {
-                  value: "true",
+                  value: "term",
```

Comment:
I'd recommend using `EmploymentDuration.Term` and `EmploymentDuration.Indeterminate` instead of making up your own strings to use here and in `durationSelectionToEnum`.

Reply:
like this https://github.com/GCTC-NTGC/gc-digital-talent/pull/4956/commits/30df366b54ee314d1e41325b97f0a21cf4a6782f  ?